index.js

```diff
@@ -57,10 +58,16 @@ function cookieSession (options) {
 
   debug('session options %j', opts)
 
+  var algorithm = opts.algorithm || 'sha1'
```

Comment:
It seems like when I delete this line, no tests fail. Can you add a test that will make sure this line stays intact?